libvirt/tests/src/backingchain/blockjob/blockjob_with_raw_option.py

```diff
@@ -0,0 +1,131 @@
+
+LOG = logging.getLogger('avocado.' + __name__)
+
+
+def run(test, params, env):
+    """
+    Test blockjob with --raw option
+
+    """
+    def setup_raw_list():
+        """
+        Prepare running domain and do blockcopy
+        """
+        test.log.info("TEST_SETUP1:Start vm and clean exist copy file")
+        test_obj.backingchain_common_setup(remove_file=True,
+                                           file_path=tmp_copy_path)
+
+        test.log.info("TEST_SETUP2:Do blockcopy")
+        cmd = "blockcopy %s %s %s --wait --verbose --transient-job " \
+              "--bandwidth %s " % (vm_name, dev, tmp_copy_path, bandwidth)
+        virsh_session = virsh.VirshSession(virsh_exec=virsh.VIRSH_EXEC,
+                                           auto_close=True)
+        virsh_session.sendline(cmd)
+
+    def test_raw_list():
+        """
+        Do blockjob with raw option.
+        1) Do blockcopy.
+        2) Do blockjob with raw option twice and check cur value is changing
+        3) Execute with --pivot and check raw option return empty
+        """
+        if not libvirt.check_blockjob(vm_name, dev, "progress", "100"):
+            _check_cur()
+            _abort_job()
+        else:
+            test.error('Blockcopy finished too fast, unable to check raw result,\
+             Please consider to reduce bandwith value to retest this case.')
+
+    def teardown_raw_list():
+        """
+        After blockcopy, clean file
+        """
+        # If some steps are broken by accident, execute --abort to
+        # stop blockcopy job
+        virsh.blockjob(vm_name, dev, options=' --abort', debug=True,
+                       ignore_status=True)
+
+        test_obj.clean_file(tmp_copy_path)
+        bkxml.sync()
+
+    def _check_cur():
+        """
+        Do twice blockjob --raw after blockcopy, and check the value of cur is
+        changed according to the progress of blockcopy.
+        """
+        test.log.info("TEST_STEP1: Do blockjob with --raw and check cur changed")
+        res_1 = virsh.blockjob(vm_name, dev, options=test_option, debug=True,
+                               ignore_status=False)
+        cur1 = libvirt_misc.convert_to_dict(res_1.stdout_text.strip(),
+                                            pattern=r'(\S+)=(\S+)')['cur']
+        time.sleep(1)
+        res_2 = virsh.blockjob(vm_name, dev, options=test_option, debug=True,
+                               ignore_status=False)
+        cur2 = libvirt_misc.convert_to_dict(res_2.stdout_text.strip(),
+                                            pattern=r'(\S+)=(\S+)')['cur']
+        LOG.debug('cur1 is %s , cur2 is %s', cur1, cur2)
+
+        if int(cur1) >= int(cur2):
+            test.fail('Two times cur value is not changed according'
+                      ' to the progress of blockcopy.')
+
+    def _abort_job():
+        """
+         Abort job and check abort success, and check no output after abort.
```

Comment:
What about changing like below?
```suggestion
        Abort the job and check if it's aborted successfully
```

Reply:
done